aggregation_mode/src/aggregators/sp1_aggregator.rs

```diff
@@ -13,25 +13,68 @@ const CHUNK_PROGRAM_ELF: &[u8] =
 const USER_PROOFS_PROGRAM_ELF: &[u8] =
     include_bytes!("../../aggregation_programs/sp1/elf/sp1_user_proofs_aggregator_program");
 
+#[allow(dead_code)]
```

Comment:
This seems like a missing feature flag, ignoring the dead code looks wrong

```suggestion
#[cfg(feature = "prove")]
```